app/com/linkedin/drelephant/tuning/hbt/FitnessManagerHBT.java

```diff
@@ -115,8 +115,7 @@ protected void calculateAndUpdateFitness(JobExecution jobExecution, List<AppResu
       }
 
       logger.info(
-          " Calculating Fitness for " + jobSuggestedParamSet.id + " " + jobSuggestedParamSet.fitnessJobExecution.id
-              + " " + jobExecution.id);
```

Comment:
Removing the reference to `jobSuggestedParamSet.fitnessJobExecution.id` fixed the `java.lang.VerifyError: Stack map does not match the one at exception handler` error.

@pralabhkumar, does the updated log message look ok to you?